src/OrchardCore.Modules/OrchardCore.Users/Services/DefaultExternalLoginUserToRelateFinder.cs

```diff
@@ -0,0 +1,42 @@
+
+using System.Runtime.CompilerServices;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using OrchardCore.Users.Abstractions;
+
+namespace OrchardCore.Users.Services;
+
+public class DefaultExternalLoginUserToRelateFinder : IExternalLoginUserToRelateFinder
+{
+  private readonly UserManager<IUser> _userManager;
+
+  public DefaultExternalLoginUserToRelateFinder(UserManager<IUser> userManager)
+  {
+    _userManager = userManager;
+  }
+
+  public bool CanManageThis(string extLoginKind)
```

Comment:
Not sure that this method is helpful

Reply:
need this method to choose the service by the External Login kind or by another application requirement